src/Webapi/Dom/Webapi__Dom__HtmlFormControlsCollection.res

```diff
@@ -0,0 +1,46 @@
+/**
+ * Spec: https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#htmlformcontrolscollection
+ */
+type t = Dom.htmlFormControlsCollection
+
+include Webapi__Dom__HtmlCollection.Impl({
+  type t = t
+})
+
+type t_namedItem = [
+  | #RadioNodeList(Dom.radioNodeList)
+  | #Button(Dom.htmlButtonElement)
+  | #FieldSet(Dom.htmlFieldSetElement)
+  | #Input(Dom.htmlInputElement)
+  | #Object(Dom.htmlObjectElement)
+  | #Output(Dom.htmlOutputElement)
+  | #Select(Dom.htmlSelectElement)
+  | #TextArea(Dom.htmlTextAreaElement)
+];
+
+let isRadioNodeList: 'a => bool = %raw(`
+  function(x) { return x instanceof RadioNodeList; }
```

Comment:
This will run into the same owner document issues if a from i created in another document and added to the current one.

Reply:
made this use the same constructor test as the `ofElement` PR